src/main/java/ttt/players/PlayerFactory.java

```diff
@@ -9,51 +8,54 @@
 
 public class PlayerFactory {
 
-    public static List<Player> create(int type, int boardSize, Input inputType) {
+    private Player humanplayer1;
+    private Player humanplayer2;
+
+    public PlayerFactory(Player player1, Player player2) {
+        humanplayer1 = player1;
+        humanplayer2 = player2;
+    }
+
```

Comment:
Client of player factory now decides what human player should be used to create games.